web/modules/custom/lecapi/tests/src/LecapiTestBase.php

```diff
@@ -52,4 +85,41 @@ protected function config($name) {
     return $this->container->get('config.factory')->getEditable($name);
   }
 
+  /**
+   * Create a customer.
+   *
+   * @return \Drupal\user\UserInterface
+   *   A new user entity.
+   */
+  protected function getCustomer() {
```

Comment:
can we rename this function to createNewCustomerUser ?

Reply:
It would be redundant because these tests should always create a new user.